dynatrace/environment_v2/settings.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
+from dynatrace.dynatrace_object import DynatraceObject
+from typing import List, Optional, Union, Dict, Any
+
+from dynatrace.http_client import HttpClient
+from dynatrace.pagination import PaginatedList
+import json
+import logging
+from http.client import HTTPConnection  # py3
+
+
+class SettingService:
+    ENDPOINT = "/api/v2/settings/objects"
+
+    def __init__(self, http_client: HttpClient):
+        self.__http_client = http_client
+    
+    def list(self,schema_id: Optional[str] = None,
+             scope: Optional[str] = None,external_ids: Optional[str] = None,
+             fields: Optional[str] = None,
+             filter:Optional[str] = None, sort:Optional[str] = None, page_size:Optional[str] = None) -> PaginatedList["DynatraceObject"]:
+        """Lists settings
+
+        :return: a list of settings with details
+        """
+        params = {
+            "schemaIds": schema_id,
+            "scope": scope,
+            "fields": fields,
+            "externalIds": external_ids,
+            "filter": filter,
+            "sort": sort,
+            "pageSize": page_size,
+        }
+        return PaginatedList(Settings, self.__http_client, target_url=self.ENDPOINT, list_item="items", target_params=params)
+    
+    def post(self,external_id,object_id,schema_id,schema_version,scope, value,validate_only):
```

Comment:
Similar to above the name should clarify what we're interacting with as well as we want to name them after the action not the HTTP method.

e.g. `create_object`

Reply:
renamed methods